art/actors/structures/xion/royal_yurt_charioth_pivot.xml

```diff
@@ -5,10 +5,7 @@
       <mesh>structural/xion_royal_yurt_charioth_pivot.dae</mesh>
       <props>
         <prop actor="fauna/zebu.xml" attachpoint="zebu_a" maxheight="20" minheight="-20"/>
-        <prop actor="fauna/zebu_b.xml" attachpoint="zebu_b" maxheight="20" minheight="-20"/>
```

Comment:
This is wrong. With your change you'll have only two zebus, instead of five.
You need to re-add zebu_a, zebu_b, zebu_c files.